README.md

```diff
@@ -1 +1,14 @@
-# nhsuk-tools-chart-components
\ No newline at end of file
+# NHS.UK Tools Chart components
+
+This repository contains the code for NHS.UK React components - a port of the [NHS.UK Tools Chart components](https://github.com/NHSDigital/nhsuk-tools-chart-components).
```

Comment:
we should expand on this and give a brief explanation on where they originated from, what their original purpose was and the goal of moving them here

Reply:
updated